monai/networks/utils.py

```diff
@@ -310,3 +312,76 @@ def train_mode(*nets: nn.Module):
         # Return required networks to eval_list
         for n in eval_list:
             n.eval()
+
+
+def compatible_mod_state(
```

Comment:
I agree that the name isn't immediately obvious to me. `copy_state_dict` sounds better, but given that it can now also do in-place, something like `copy_model_state` might be better.